// @copilotkit/react-ui/dist/components/index.mjs
import "../chunk-MMVDU6DF.mjs";
import "../chunk-PXEVB7IK.mjs";
import {
  CopilotPopup
} from "../chunk-ECCHEFQV.mjs";
import {
  CopilotSidebar
} from "../chunk-KCV67665.mjs";
import "../chunk-WB3YULQ4.mjs";
import "../chunk-WN46UIN3.mjs";
import "../chunk-TIDV74OE.mjs";
import "../chunk-UH2UFL5W.mjs";
import "../chunk-V7W6IM2V.mjs";
import {
  CopilotDevConsole
} from "../chunk-VGPQYMKJ.mjs";
import "../chunk-Q5V6S67N.mjs";
import {
  shouldShowDevConsole
} from "../chunk-6TCUJ3B7.mjs";
import "../chunk-KXE2JCUH.mjs";
import "../chunk-NRA3CFEE.mjs";
import "../chunk-BH6PCAAL.mjs";
import "../chunk-UFN2VWSR.mjs";
import {
  CopilotChat
} from "../chunk-JZAMNIUD.mjs";
import "../chunk-DSQGQJI4.mjs";
import "../chunk-24TDU7MY.mjs";
import {
  RenderImageMessage
} from "../chunk-LUPGADWY.mjs";
import "../chunk-XFCMZH2H.mjs";
import "../chunk-RVLCPPEL.mjs";
import {
  AssistantMessage
} from "../chunk-OZXUB3V7.mjs";
import {
  UserMessage
} from "../chunk-HWMFMBJC.mjs";
import "../chunk-XZNY26GH.mjs";
import "../chunk-PLHTVHUW.mjs";
import "../chunk-EMIYIMQ6.mjs";
import "../chunk-CGEAG65D.mjs";
import "../chunk-YQFVRDNC.mjs";
import {
  Markdown
} from "../chunk-ABHUX6T6.mjs";
import "../chunk-KENCH7RN.mjs";
import "../chunk-IIRQWGWV.mjs";
import {
  useChatContext
} from "../chunk-IEMQ2SQW.mjs";
import "../chunk-XWG3L6QC.mjs";
import "../chunk-54JAUBUJ.mjs";
import "../chunk-JGMFJZMG.mjs";
import "../chunk-S5MBUNGN.mjs";
import "../chunk-MRXNTQOX.mjs";
export {
  AssistantMessage,
  CopilotChat,
  CopilotDevConsole,
  CopilotPopup,
  CopilotSidebar,
  Markdown,
  RenderImageMessage,
  UserMessage,
  shouldShowDevConsole,
  useChatContext
};
